mesheryctl/pkg/utils/helpers.go

```diff
@@ -70,6 +71,8 @@ var (
 	// DockerComposeFile is the default location within the MesheryFolder
 	// where the docker compose file is located.
 	DockerComposeFile = "meshery.yaml"
+	// ManifestsFolder is where the Kubernetes manifests are stored
+	ManifestsFolder = "manifests"
```

Comment:
create new folder in `~/.meshery` called `manifests` to download all the manifest files.